deebot_client/commands/json/map.py

```diff
@@ -228,11 +205,19 @@ def _handle_body_data_dict(
             elif subtype:
                 name = cls._ROOM_NUM_TO_NAME.get(subtype, None)
 
+            coordinates: str | None = None
+            if data.get("compress", 0) == 1:
+                # NOTE: newer bot's return coordinates as base64 decoded string
+                coordinates = decompress_7z_base64_data(data["value"]).decode()
+            if coordinates is None:
+                # NOTE: older bot's return coordinates direct as comma separated list
+                coordinates = data["value"]
+
```

Comment:
Is this change required when you created `GetMapSetV2`?

Reply:
It has been a while since then, and I'm not entirely certain if I saw both possibilities at that point. This is why I decided to verify if compression was included, because i think i saw it is possible that some are compressed and some not. 

If you require a recheck, I can conduct some tests in the next few days.

Reply:
If you have time to recheck that would be nice :)
As far as I know only V2 should support compression

Reply:
Removed.

I quick checked it and i think there should be no situation where this is needed,
as `MapSubsetEvent` is for `v2` notified in `GetMapSetV2`.
Possible i had problems at the beginnings because of the error described below, 
with remove of `v2` inside `deebot_client/messages/__init__.py`.